specs/protocol/README.md

```diff
@@ -422,25 +419,38 @@ Currency amounts have type `DecimalString`, which is string containing a decimal
 ### `RFQ (Request For Quote)`
 > Alice -> PFI: "OK, that offering looks good. Give me a Quote against that Offering, and here is how much USD (payin currency) I want to trade for BTC (payout currency). Here are the credentials you're asking for, the payment method I intend to pay you USD with, and the payment method I expect you to pay me BTC in."
 
-| field        | data type                                        | required | description                                                                           |
-| ------------ | ------------------------------------------------ | -------- | ------------------------------------------------------------------------------------- |
-| `offeringId` | string                                           | Y        | Offering which Alice would like to get a quote for                                    |
-| `claims`     | string[]                                         | Y        | an array of claims that fulfill the requirements declared in an [Offering](#offering) |
-| `payin`      | [`SelectedPayinMethod`](#selectedpaymentmethod)  | Y        | selected payin amount, method, and details                                            |
-| `payout`     | [`SelectedPayoutMethod`](#selectedpaymentmethod) | Y        | selected payout method, and details                                                   |
+| field          | data type                                        | required | description                                                                |
+|----------------|--------------------------------------------------|----------|----------------------------------------------------------------------------|
+| `offeringId`   | string                                           | Y        | Offering which Alice would like to get a quote for                         |
+| `claimsHashes` | string[]                                         | Y        | an array of salted digests of the claims appearing in `privateDate.claims` |
```

Comment:
```suggestion
| `claimsHashes` | string[]                                         | Y        | an array of salted hashes of the claims appearing in `privateDate.claims` |
```

Reply:
Ah, this answers [my question above](https://github.com/TBD54566975/tbdex/pull/294#discussion_r1543383343) about whether you like the term "salted digest".

Reply:
Updated to "salted hash"